site/content/posts/2021-08-27-supporting-ad-oidc-workflows.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Pinniped v0.11.0: Easy Configurations for Active Directory, OIDC CLI workflows and more"
+slug: supporting-ad-oidc-workflows
+date: 2021-07-28
```

Comment:
I think you meant 2021-08-31 (which is Monday)

Reply:
Yes resolved this in new changes